eng/pipelines/common/variables.yml

```diff
@@ -15,6 +15,10 @@ variables:
   value: ${{ and(eq(variables['System.TeamProject'], 'public'), ne(variables['Build.Reason'], 'PullRequest')) }}
 - name: isNotFullMatrix
   value: ${{ and(eq(variables['System.TeamProject'], 'public'), eq(variables['Build.Reason'], 'PullRequest')) }}
+- name: isNotManualAndIsPR
+  value: ${{ and(not(in(variables['Build.DefinitionName'], 'manual-runtime-staging', 'manual-runtime')), eq(variables['System.TeamProject'], 'public'), eq(variables['Build.Reason'], 'PullRequest')) }}
+- name: isManualOrIsNotPR
```

Comment:
I'd call the variable `isReducedFullMatrix` since we're essentially running a bit less than the normal fullmatrix run, I don't think we need to tie this into PR vs. rolling vs. manual.

Reply:
alternatively, I wonder if we should instead just set isFullMatrix for the manual pipeline and only update the few places where we really really don't want to run on PRs even in the manual pipeline.

Reply:
> alternatively, I wonder if we should instead just set `isFullMatrix` for the manual pipeline and only update the few places where we really really don't want to run on PRs even in the manual pipeline.

For runtime-staging that might be ok, but I think it would be harder to work around in runtime.  That's why I thought Santi's suggestion for another variable was a good one.

Reply:
> I'd call the variable `isReducedFullMatrix` since we're essentially running a bit less than the normal fullmatrix run, I don't think we need to tie this into PR vs. rolling vs. manual.

I thought Ankit's suggestion was easier to work with when modifying the yml.  Honestly, I don't care what we name it.

@radical @safern, thoughts?

Reply:
When I quickly looked it didn't seem used in a lot of relevant places but ok :)

That actually raises another question, or maybe I'm misunderstanding the conditions: we're running _all_ of the legs in the manual pipeline, even those that already run as part of the normal pipeline right? that seems suboptimal and a waste of resources.

Reply:
We'll only run a subset of the legs.  Basically, all the ones that are configured to run smoke tests by default.

Reply:
My biggest concern with using isFullMatrix is for runtime.yml because that runs as a rolling build which has a LOOT of tests and also a lot of those would already be covered by the default run we get on PRs, so that's why I suggested a new variable.

Reply:
@akoeplinger @radical @safern Are the `isNotManualAndIsPR` and `isManualOrIsNotPR` variables ok or do we want a different name?  My vote is to go with them, but I'd like your opinions before moving forward.

Reply:
I would prefer sticking with them, but would be fine with whatever @safern, and @akoeplinger agree upon.

Reply:
I'm fine with sticking with them too.

Reply:
Fine with sticking with those names as well.